src/datasets.c

```diff
@@ -593,11 +609,45 @@ void DatasetPostReloadCleanup(void)
     SCMutexUnlock(&sets_lock);
 }
 
+static void GetDefaultMemcap(uint64_t *memcap, uint32_t *hashsize) // TODO Do something about this name! OR Save default memcap in dataset struct?
+{
+    ConfNode *datasets = ConfGetNode("datasets");
+    if (datasets != NULL) {
```

Comment:
nit/style: If `datasets == NULL`, return. This will remove one level of indentation